src/catalog/src/local/manager.rs

```diff
@@ -513,7 +526,14 @@ impl CatalogManager for LocalCatalogManager {
 
         let mut sys_table_requests = self.system_table_requests.lock().await;
         sys_table_requests.push(request);
-
+        increment_gauge!(
+            crate::metrics::METRIC_CATALOG_MANAGER_TABLE_COUNT,
+            1.0,
+            &[(
+                crate::metrics::METRIC_DB_LABEL,
+                build_db_string(&catalog_name, &schema_name)
+            )],
```

Comment:
How about providing a method to return a (key, value) pair?

Reply:
resolved